index.bs

```diff
@@ -109,20 +109,26 @@ g g:hover text {
 <!--[if lt IE 9]><script src='undefined://www.w3.org/2008/site/js/html5shiv.js'></script><![endif]-->
 
 <h2 class="no-num no-toc no-ref" id="pp">
-Relation of Process Document to Patent Policy</h2>
+Relation of Process Document to Patent Policy and Other Policies</h2>
 
 	<!-- DO NOT REMOVE THIS SECTION OR ITS REFERNCE TO THE PATENT POLICY WITHOUT DISCUSSING WITH PSIG / COUNSEL. -->
 
 	W3C Members' attention is called to the fact
 	that provisions of the Process Document are binding on Members
 	per the <a href="https://www.w3.org/Consortium/Agreement/Member-Agreement">Membership Agreement</a> [[MEMBER-AGREEMENT]].
 	The <a href="https://www.w3.org/Consortium/Patent-Policy/">W3C Patent Policy</a> [[!PATENT-POLICY]]
-	is incorporated by normative reference as a part of the Process Document,
-	and is thus equally binding.
-
-	The Patent Policy places additional obligations on Members, Team, and other participants in W3C.
+	and other policies
+	incorporated by normative reference as a part of the Process Document
+	are thus equally binding.
+	These policies are written with the intent that there be no conflict,
+	but in the event that one exists,
+	the Member Agreement takes precedence over the Patent Policy,
+	which takes precedence over this Process,
+	which takes precedence over any <a href="#normative">other normative references</a>.
```

Comment:
The link indicates this means "other normative references" _from this Process_ but that's a subtle way of conveying the information. Consider "other normative references from this Process" as a clearer alternative?